From a4b093f7366fdb429ca1781144d3985fa50d0fbb Mon Sep 17 00:00:00 2001
From: Julien Grall <julien.grall@linaro.org>
Date: Tue, 17 Mar 2015 16:00:34 +0000
Subject: [PATCH] libfdt: Add missing functions to shared library

The commit 4e76ec7 "libfdt: Add fdt_next_subnode() to permit easy
subnode iteration" adds new functions (fdt_{first,next}_subnode) but
forgot to mark them as 'global' in the shared library.

Signed-off-by: Julien Grall <julien.grall@linaro.org>
---
 libfdt/version.lds | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libfdt/version.lds b/libfdt/version.lds
index 80b322b..941208e 100644
--- a/libfdt/version.lds
+++ b/libfdt/version.lds
@@ -54,6 +54,8 @@ LIBFDT_1.2 {
 		fdt_get_property_by_offset;
 		fdt_getprop_by_offset;
 		fdt_next_property_offset;
+		fdt_first_subnode;
+		fdt_next_subnode;
 
 	local:
 		*;
-- 
2.4.0

